src/models/swarm.model.ts

```diff
@@ -0,0 +1,14 @@
+import { Table, Column, Model } from 'sequelize-typescript'
+
+@Table({
+  freezeTableName: true,
+  tableName: 'swarm',
+  timestamps: false
+})
+export default class SwarmModel extends Model {
```

Comment:
I would choose a different name as "Swarm" does not really give me much information, also would keep it maybe IPFS-agnostics so it could be possible to reuse later on if we will be implementing Swarm (heh, see a collision of names :-D ). Maybe the "DirectAddress" could be an option?